src/features/view-subdomains/SubdomainTable/SubdomainTable.test.tsx

```diff
@@ -0,0 +1,214 @@
+import { SubdomainTable } from './SubdomainTable';
```

Comment:
Import React to prevent - 

<img width="980" alt="Screenshot 2023-03-11 at 09 17 09" src="https://user-images.githubusercontent.com/39112648/224438517-68a5d18c-3603-465a-97af-3dc224abd9dd.png">

Reply:
Good shout!